src/util/git-client/git-client.ts

```diff
@@ -1,28 +1,12 @@
-import { context } from '@actions/github';
 import { spawnSync } from 'child_process';
 
-export const getPullRequestNumber = (): number => {
-  return context.payload.pull_request?.number || 0;
-};
-
-export const isMainBranch = (): boolean => {
-  return context.ref === 'refs/heads/main';
-};
-
 export const getTagSha = (tagName: string): string => {
   return spawnSync('git', ['rev-list', '-n', '1', tagName]).stdout.toString().trim();
 };
 
 export const isChangeInPath = (target: string, path: string): boolean => {
   const { status } = spawnSync('git', ['diff', '--quiet', `${target}...`, '--', path]);
-
-  if (status === 1) {
-    return true;
-  }
-
-  if (status === 0) {
-    return false;
-  }
-
+  if (status === 1) return true;
+  if (status === 0) return false;
```

Comment:
`return !!status` maybe it work also?

Reply:
Yes it would work, but the fallowing line will never be executed.